code/game/objects/items/storage/backpack.dm

```diff
@@ -292,15 +289,40 @@
 			select_gamemode_skin(type)
 		..()
 
+/obj/item/storage/backpack/marine/standard
+	name = "\improper lightweight IMP backpack"
+	desc = "The standard-issue pack of the USCM forces. Designed to slug gear into the battlefield."
+	bypass_w_limit = list("/obj/item/device/m56d_gun",
+					"/obj/item/ammo_magazine/m56d",
+					"/obj/item/device/m56d_post",
+					"/obj/item/device/turret_top",
+					"/obj/item/device/turret_sensor",
+					"/obj/item/ammo_magazine/sentry",
+					)
+
 /obj/item/storage/backpack/marine/medic
 	name = "\improper USCM medic backpack"
 	desc = "The standard-issue backpack worn by USCM medics."
 	icon_state = "marinepackm"
+	bypass_w_limit = list("/obj/item/device/m56d_gun",
```

Comment:
Medic backpacks don't need this.

Reply:
All backpacks should have this capability.

Reply:
Not really.

Reply:
Yes. No need for all these ridiculous arbitrary differences, and it's one at least semi-compelling reason to have a backpack; sure it's generally not going to be a thing for medics, but PFCs are often on M56D duty at the least.

Reply:
Think about adding something medical related for the medic backpacks instead (no large containers such as medkits, please).

Reply:
Added a defibrillator recharge unit.